Content.Client/Stylesheets/StyleNano.cs

```diff
@@ -615,27 +621,40 @@ public StyleNano(IResourceCache resCache) : base(resCache)
                     .Prop(Label.StylePropertyFont, notoSans12),
 
                 // Thin buttons (No padding nor vertical margin)
-                Element<EntityContainerButton>().Class(StyleClassStorageButton)
+                Element<ContainerButton>().Class(StyleClassStorageButton)
                     .Prop(ContainerButton.StylePropertyStyleBox, buttonStorage),
 
-                Element<EntityContainerButton>().Class(StyleClassStorageButton)
+                Element<ContainerButton>().Class(StyleClassStorageButton)
                     .Pseudo(ContainerButton.StylePseudoClassNormal)
                     .Prop(Control.StylePropertyModulateSelf, ButtonColorDefault),
 
-                Element<EntityContainerButton>().Class(StyleClassStorageButton)
+                Element<ContainerButton>().Class(StyleClassStorageButton)
                     .Pseudo(ContainerButton.StylePseudoClassHover)
                     .Prop(Control.StylePropertyModulateSelf, ButtonColorHovered),
 
-                Element<EntityContainerButton>().Class(StyleClassStorageButton)
+                Element<ContainerButton>().Class(StyleClassStorageButton)
                     .Pseudo(ContainerButton.StylePseudoClassPressed)
                     .Prop(Control.StylePropertyModulateSelf, ButtonColorPressed),
 
-                Element<EntityContainerButton>().Class(StyleClassStorageButton)
+                Element<ContainerButton>().Class(StyleClassStorageButton)
                     .Pseudo(ContainerButton.StylePseudoClassDisabled)
                     .Prop(Control.StylePropertyModulateSelf, ButtonColorDisabled),
 
-                // action slot hotbar buttons
-                new StyleRule(new SelectorElement(typeof(ActionSlot), null, null, new[] {ContainerButton.StylePseudoClassNormal}), new[]
+                // These are for the weight indicator on the storage buttons
+                Element<Label>().Class(StyleClassStorageWeightLabel)
```

Comment:
Should probably be done with Pseudo Classes like the buttons above on line 639.

Reply:
The pseudo styles are all set internally by each control (e.g., hover, pressed, disabled for buttons). I can't set it externally without creating a new Label control. 

While it would be nice to have some sort of label that auto-resizes its font based on whether the text fits in some restricted area, that should probably be done in a separate engine PR.

Reply:
This is the label under the Texture text so it's PseudoClass shouldn't get changed by the button.

Reply:
Oh whoops, I see what you mean. Nevermind.